docs/src/eeg.md

```diff
@@ -9,20 +9,20 @@ TopoPlots.eeg_topoplot
 
 
 
-So for the standard 10/20 montage, one can drop the `positions` attribute:
+For the standard 10/20 montage, one can drop the `positions` attribute:
 ```@example 1
 using TopoPlots, CairoMakie
 
 labels = TopoPlots.CHANNELS_10_20
-TopoPlots.eeg_topoplot(rand(19), labels; axis=(aspect=DataAspect(),), label_text=true, label_scatter=(markersize=10, strokewidth=2,))
+TopoPlots.eeg_topoplot(rand(19); labels=labels, axis=(aspect=DataAspect(),), label_text=true, label_scatter=(markersize=10, strokewidth=2,))
```

Comment:
```suggestion
TopoPlots.eeg_topoplot(rand(19); labels, axis=(aspect=DataAspect(),), label_text=true, label_scatter=(markersize=10, strokewidth=2,))
```


for `f(; kwarg=kwarg)`, you can just write `f(; kwarg)` (but the semicolon is necessary)

Reply:
i know it, but never sure whether to use that syntax sugar or not